### ISSUE TITLE
Starting the game from launcher sometimes displays that you have been disconnected and should relaunch the game

### ISSUE THREAD

**Issue by culinko** [HearthSim/hs-bugs#442]
Sometimes when you start the game from launcher, it will tell you that you have been disconnected from the Battle.net service and you should relaunch the game in order to play. This started happening a couple of patches ago and the cause is unknown yet. Some people mentioned running the client in windowed mode and in background (unfocused) when it happens.
* https://www.reddit.com/r/hearthstone/comments/4df6a8/why_do_i_have_to_open_the_game_twice_for_it_to/

**Comment by Martios** [comment]
Is this still happening? After the latest patch this isn't happening to me anymore, while before the patch it always happened.

**Comment by culinko** [comment]
Yeah it's not happening for me anymore but I will still wait a bit until I close it, because it might return.

**Comment by culinko** [comment]
I reproduced this in current patch just now.
